src/app/portlets/dot-apps/dot-apps-list/dot-apps-list-resolver.service.ts

```diff
@@ -2,7 +2,14 @@ import { Observable } from 'rxjs';
 import { Injectable } from '@angular/core';
 import { Resolve, RouterStateSnapshot, ActivatedRouteSnapshot } from '@angular/router';
 import { DotLicenseService } from '@services/dot-license/dot-license.service';
-import { take } from 'rxjs/operators';
+import { map, mergeMap, take } from 'rxjs/operators';
+import { DotApps } from '@shared/models/dot-apps/dot-apps.model';
+import { DotAppsService } from '@services/dot-apps/dot-apps.service';
+
+export interface DotAppsListResolverData {
```

Comment:
Don't we have a model folder for this interfaces?

Reply:
done!